views/index.erb

```diff
@@ -1,8 +1,31 @@
 <h1><%= @title %></h1>
 
 Output generated: <%= Time.now %>
+
 <pre>
 $ <%= @command %>
-
-<%= @output %>
 </pre>
+
+<table class="table table-bordered">
+  <tr>
+    <th>USER</th>
+    <th>PID</th>
+    <th>%CPU</th>
+    <th>%MEM</th>
+    <th>VSZ</th>
+    <th>RSS</th>
+    <th>TTY</th>
+    <th>STAT</th>
+    <th>START</th>
+    <th>TIME</th>
+    <th>COMMAND</th>
+  </tr>
+  <% @output.each do |row| %>
+  <tr>
+    <% row.each do |value| %>
+      <td><%= value %></td>
+    <% end %>
+  </tr>
```

Comment:
I think my concern is that whatever polish we provide to this app in form of a tutorial, that is the level of polish that the resulting modified apps will have.

Also, our example app is from "ps" to "quota" => and the example is not to put the output of two commands into a  `<pre>` and call it done, is it?

My assumption is that an OOD app that is a dashboard style app would always follow the pipeline: `run command => parse output => visualize` instead of `run command => display output`.

Thus the idea that we go from  `run ps command => parse ps output => visualize` to  `run quota command => parse quota output => visualize` (or some derivative).

I guess we could have `run ps command => display output in pre` and move to `run quota command => parse quota output => visualize` but then that second will have to take some form that is easy to read and understand but not promote bad practices.

Why is 

```ruby
ps_string.scan(/(([^\s]+\s+){10})(.*)/).map { |p| AppProcess.new(*(p.first.split), p.last) }
```

worse than

```ruby
  ps_string.scan(/([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+(.*)/).map { |p| AppProcess.new(*p) }
```